scripts/version

```diff
@@ -12,8 +15,13 @@ GIT_TAG=$(git tag -l --contains HEAD | head -n 1)
 
 if [[ -z "$DIRTY" && -n "$GIT_TAG" ]]; then
     VERSION=$GIT_TAG
+    # Search for minimum upgradable version.
+    # If the release does not come with a corresponding minimum upgradable version, the build will error out.
+    MIN_UPGRADABLE_VERSION=$(yq -e e ".versions | map(select(.name == \"$VERSION\")) | .[0].minUpgradableVersion" "${PACKAGE_DIR}"/upgrade-matrix.yaml)
```

Comment:
It would be nice to print out `MIN_UPGRADABLE_VERSION` at the end of the file.

Reply:
Updated it, thanks.